avatars/avatars.py

```diff
@@ -30,10 +31,11 @@ class BaseAvatar(object):
     Base class for all avatar implementations.
     """
 
-    def __init__(self, preferred_url_scheme, colors, http_client):
+    def __init__(self, preferred_url_scheme, colors, http_client, feature_fips):
```

Comment:
Can you set `feature_fips` default to false?

Reply:
Actually, you don't really need this. Since your config flag starts with `FEATURE_`, you could just `import features` and check with:
```
import features
...

features.fips
```

Reply:
Just updated PR to use features module instead of manually passing the value.